mappings/net/minecraft/network/ClientConnection.mapping

```diff
@@ -29,6 +29,7 @@ CLASS none/eq net/minecraft/network/ClientConnection
 	FIELD p disconnected Z
 	METHOD <init> (Lnone/fm;)V
 		ARG 0 side
+	METHOD a tick ()V
```

Comment:
I'm ok with `update` but it's literally just called by server tick. If you look at the method body everything called is `tick` as well.